Makefile

```diff
@@ -20,6 +20,9 @@ testD:
 testE:
 	make VERBOSE=$(VERBOSE) NODIFF=$(NODIFF) DEBUG=$(DEBUG) OPTDUMP=$(OPTDUMP) OPTS="ignoreclosebraces" -C test test
 
+doc-container: zinit.zsh zinit-side.zsh zinit-install.zsh zinit-autoload.zsh
```

Comment:
Can we extract: 

```
zinit.zsh zinit-side.zsh zinit-install.zsh zinit-autoload.zsh
```

into variable to avoid duplicating it in `doc` target?

Reply:
Maybe? I don't know how :D

Reply:
No worries, just a nit :^)